### ISSUE TITLE
[FEATURE]: Add Separate Edit Feature for Editing Profile Details[with add Other Details]

### ISSUE THREAD

**Issue by NarendraKoya999** [narayan954/dummygram#462]
## Issue Description:
-  **Go To This 'Dummygram' Website.**
- **Signing with Either Google OR Enter New E-Mail and Password Details.**
- **After Entering into the Main Page, Click on Profile Option of Left Side Bar.**
- **Observe Full Profile Card. In this Profile Card, Only Possible to Edit Profile Picture of Particular User.**
- **Also, Along with User Name Shown on the Profile Card . I want to suggest Respective E-Mail, Password(Encrypted) Detail and Add Save , Cancel  Buttons Add in Respective Profile Card Only When Click on Edit Option.**

> Where To Add This Edit Icon 
- **This Edit Icon adding to Right Side of Showing User Name. Also, all above Mentioned like Combination of User Name and to add E-Mail, Password(Encrypted), Save , Cancel Button Options.**
- **When Clicks on Edit Option, Special Screen Whatever Edit the Profile Details Options and Clickable Save OR Cancel Buttons.**
- **Only Allow This Edit Option for the Whole User Details (User Name, E-Mail, Password Etc).**

### Screenshot of My Feature
<img width="512" alt="dummygram-1" src="https://github.com/narayan954/dummygram/assets/84512164/c17f0c5d-aa70-4812-86e5-c4dddf3cc32f">

### NOTE: Suggested This Feature More Attractable and It's also Corelated to Latest Design Trends.

**Comment by narayan954** [comment]
@NarendraKoya999 can you please recheck if similar issue exists already. I think this issue exists, you can try to claim it. If it doesn't, mention me and I'll assign you :)

**Comment by NarendraKoya999** [comment]
Hi @narayan954  Existing Issue No #284 raised For making User Post card Has Edit and Delete options . But, my Issue Only Suggest Profile Details Card has to Implement Edit Option, Save , Cancel Options. Please check Once .

.

**Comment by narayan954** [comment]
> Hi @narayan954 Existing Issue No #284 raised For making User Post card Has Edit and Delete options . But, my Issue Only Suggest Profile Details Card has to Implement Edit Option, Save , Cancel Options. Please check Once .
> 
> .

Alright, you can work on this

**Comment by NarendraKoya999** [comment]
Please Assign this Issue to Others

**Comment by narayan954** [comment]
> Please Assign this Issue to Others

yeah sure :)

**Comment by vk20july** [comment]
Can i work on this issue under gssoc'23

**Comment by narayan954** [comment]
> Can i work on this issue under gssoc'23

Yes sure, go ahead

**Comment by vk20july** [comment]
@narayan954 can you explain the issue in simple way?

**Comment by narayan954** [comment]
> * Respective E-Mail, Password(Encrypted) Detail and Add Save , Cancel  Buttons Add in Respective Profile Card Only When Click on Edit Option.

issue is to basically add respective E-Mail, password(Encrypted) detail and Add Save , Cancel Buttons in Respective Profile Card Only When Click on Edit Option.


More detail:
This Edit Icon adding to Right Side of Showing User Name. Also, all above Mentioned like Combination of User Name and to add E-Mail, Password(Encrypted), Save , Cancel Button Options.
When Clicks on Edit Option, Special Screen Whatever Edit the Profile Details Options and Clickable Save OR Cancel Buttons.
Only Allow This Edit Option for the Whole User Details (User Name, E-Mail, Password Etc).

**Comment by NarendraKoya999** [comment]
Hi @narayan954 . If Possible, I will take up the Issue. Because of It's Still Open ??

**Comment by narayan954** [comment]
> Hi @narayan954 . If Possible, I will take up the Issue. Because of It's Still Open ??

profile page is being updated, please wait a day so you can work on the new profile page...

**Comment by NarendraKoya999** [comment]
Ok Sure @narayan954

**Comment by NandiniGera** [comment]
hi @narayan954, can I take this issue if it's still open?

**Comment by narayan954** [comment]
> hi @narayan954, can I take this issue if it's still open?

yeah sure, if @NarendraKoya999 isn't working already. Narendra do let us know if you're working on this issue?

**Comment by NarendraKoya999** [comment]
> > hi @narayan954, can I take this issue if it's still open?
> 
> yeah sure, if @NarendraKoya999 isn't working already. Narendra do let us know if you're working on this issue?

Sorry @narayan954 . Due to some Reasons, I making these changes not possible for me . Please give this issue to another Person

**Comment by narayan954** [comment]
> > > hi @narayan954, can I take this issue if it's still open?
> > 
> > 
> > yeah sure, if @NarendraKoya999 isn't working already. Narendra do let us know if you're working on this issue?
> 
> Sorry @narayan954 . Due to some Reasons, I making these changes not possible for me . Please give this issue to another Person

all cool man! :)

**Comment by narayan954** [comment]
> hi @narayan954, can I take this issue if it's still open?

you can go ahead!
